docs/how-to-test-translations-locally.md

```diff
@@ -0,0 +1,133 @@
+# How to Test Translations Locally
+
+> [!NOTE]
+> This process is not required, but documented in case you would like to preview what your translations will look like.
+
+If you would like to test your translations on a local instance of the freeCodeCamp `/learn` site, first ensure you have [set up the codebase](how-to-setup-freecodecamp-locally.md).
+
+## Enabling a Language
+
+There are a few steps to take in order to allow the codebase to build in your desired language.
+
+First, visit the `config/i18n/all-langs.js` file to add the language to the available languages list and configure the values. There are four objects here.
+
+- `availableLangs`: For both the `client` and `curriculm` arrays, add the text name of the language. This is the value that will be used in the `.env` file later.
```

Comment:
```suggestion
- `availableLangs`: For both the `client` and `curriculum` arrays, add the text name of the language. This is the value that will be used in the `.env` file later.
```

Reply:
I promise I'll learn how to spell one day. 🙂